students/JaNiece/goals.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="index.css">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> My Webpage </title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <img src="images/sm.jpeg" class="logo">
+      <a href="index.html"> Home </a>
+      <a href="life.html"> Learn More </a>
+      <a href="goals.html"> My Goals </a>
+      <a href="contact.html" class= "right"> Contact </a>
+    </nav>
+  </header>
+
+  <section class="goals">
+    <h2> My Goals </h2>
+    <p>
+      Sometimes it's good to have your goals aligned so you can motivate yourself.
+      In my case, my goals are:
```

Comment:
🙃 Keep in mind if you create a new line in your HTML source code, it doesn't create a line break in the browser (i.e. this will end up as one paragraph). So typically you'd not add the extra line between "yourself." and "In my case" since it's going to collapse in the browser anyway. Another way of looking at this: if you're wanting a line break / new paragraph, you have to explicitly add them in the HTML.